analytics/views.py

```diff
@@ -1011,6 +1014,36 @@ def get_activity(request: HttpRequest) -> HttpResponse:
         context=dict(data=data, title=title, is_home=True),
     )
 
+@require_server_admin
+def get_info(request: HttpRequest) -> HttpResponse:
+    context = {}
+    if request.method == "POST":
+        realm_id = request.POST.get("realm_id", None)
+        new_plan_type = request.POST.get("plan_type", None)
+        realm = Realm.objects.get(id=realm_id)
+        old_plan_type = realm.plan_type
+        realm.plan_type = new_plan_type
+        realm.save(update_fields=["plan_type"])
+        context["message"] = "Plan type of {} changed to {} from {} ".format(realm.name, new_plan_type, old_plan_type)
+
+    query = request.GET.get("q", None)
+    if query:
+        key_words = get_invitee_emails_set(query)
+        users = UserProfile.objects.filter(email__in=key_words)
+        if users:
+            for user in users:
+                user.avatar_url = avatar_url(user)
+                user.realm.realm_icon_url =  realm_icon_url(user.realm)
+                user.realm.admins = UserProfile.objects.filter(realm=user.realm, is_realm_admin=True)
```

Comment:
We don't want to do a database query inside a loop -- that's going to be super slow if there are a lot of matches

Reply:
(Though this is only a problem if we're doing a more expansive search than exact matches)